config/default/kustomization.yaml

```diff
@@ -28,7 +28,6 @@ patchesStrategicMerge:
 - manager_image_patch_edited.yaml
 - manager_webhook_patch.yaml
 - webhookcainjection_patch.yaml
-- manager_prometheus_metrics_patch.yaml
```

Comment:
This won't lead to prometheus metrics not getting exposed during deployment of CAPC (without tilt). right?

Reply:
Yes it does. I've done a little round across other providers ([CAPI](https://github.com/kubernetes-sigs/cluster-api/blob/main/config/default/kustomization.yaml), [CABPK](https://github.com/kubernetes-sigs/cluster-api/blob/main/bootstrap/kubeadm/config/default/kustomization.yaml) [CAPO](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/main/config/default/kustomization.yaml) [CAPD](https://github.com/kubernetes-sigs/cluster-api/blob/main/test/infrastructure/docker/config/default/kustomization.yaml) and none of them do this in the released manifest. [CAPA](https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/main/config/default/kustomization.yaml) took a different approach and [creates a little helper service ](https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/main/config/default/metrics_service.yaml).